@app/client/package.json

```
@@ -3,8 +3,8 @@
   "private": true,
   "version": "0.0.0",
   "scripts": {
-    "build": "cd src && node -r @app/config/env ../node_modules/.bin/next build",
-    "test": "NODE_ENV=test node -r @app/config/env ./node_modules/.bin/jest"
+    "build": "cd src && npx -n \"-r @app/config/env\" next build",
+    "test": "cross-env NODE_ENV=test npx -n \"-r @app/config/env\" jest"
```

When using npx we don't want to risk it doing temporary installs of different versions just because the user's node_modules cache is out of date (or, for example, they're using yarn2 and the files don't exist where npx looks for them). Let's add the `--no-install` flag:

```suggestion
    "build": "cd src && npx --no-install -n \"-r @app/config/env\" next build",
    "test": "cross-env NODE_ENV=test npx --no-install -n \"-r @app/config/env\" jest"
```

---

(Please apply this change throughout the entire PR)